test/SSTClosGovernorTest.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2015 - 2022, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <memory>
```

Comment:
Perhaps it would be good to put all the std includes together, and all the google includes together?
```cpp
#include <memory>
#include <stdexcept>

#include "gmock/gmock-matchers.h"
#include "gmock/gmock.h"
#include "gtest/gtest.h"
```